src/client/Microsoft.Identity.Client/Internal/RequestContext.cs

```diff
@@ -24,6 +25,8 @@ internal class RequestContext
 
         public CancellationToken UserCancellationToken { get; }
 
+        public Dictionary<string, object> CacheDetails { get; } = new Dictionary<string, object>();
```

Comment:
Is it worth having `object` here and not `string`? I guess `object` is more generic, but will there ever be datapoints which are not strings?

Reply:
I kept it as object because the cache latency is logged as a long and cache used as an int. But I guess all of that can be converted to string. And convert them to long when logging as we are logging latencies as long.